plugin.xml

```diff
@@ -8,13 +8,44 @@
   </js-module>
   
   <platform name="android">
+
+    <config-file target="AndroidManifest.xml" parent="application">        
+    <activity android:name=".CustomScannerActivity"
```

Comment:
Don't you need the full package name here?

Reply:
For now it seems to be working. I will check if it is necessary to change or not with mabs in the future.